cityofaustin/atd-moped
# Implement "Project status" edit modal

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/8806

## Testing
**URL to test:** 

https://8806-status-edit-modal--atd-moped-main.netlify.app/moped/dashboard

**Steps to test:**
1. If you are not already following a project or in a project, add projects to your dashboard
2. Click on the status and the comment modal should appear (~~question, should the cursor change? its a text cursor now when you hover~~. Also, if there is no status, I'm rendering a `-` would appreciate feedback on this decision)
3. Add a status / edit a status. Upon completion, the modal should automatically close
4. your updated most recent status should be visible in the table
5. Also check that normal comments (https://8806-status-edit-modal--atd-moped-main.netlify.app/moped/projects/1827?tab=comments) still work as expected. 

My package-lock.json changed from when moment.js was added to the project and then taken away. Let me know if this should not be committed. 

---
#### Ship list
